packages/core/src/components/textarea/textarea.tsx

```diff
@@ -108,15 +109,18 @@ export class TextareaComponent {
             ref={textArea => (this.nativeTextArea = textArea)}
             class={{ disabled: this.disabled, invalid: this.invalid }}
             style={this.width ? { maxWidth: `${this.width}px` } : {}}
-            id={this.inputId}
+            id={this.textareaId}
             value={value}
             maxLength={this.maxLength}
             onInput={this.onInput}
             onBlur={this.onBlur}
+            aria-invalid={this.invalid ? 'true' : 'false'}
+            aria-describedby={(this.hint ? this.hintId : null) + ' ' + (this.invalid ? this.errorId : null)}
```

Comment:
For both the `hintId` and the `errorId` we seem to be referencing elements that don't exist / or don't have that ID assigned.

I would expect to see the hint element above have that ID, e.g.

```typescript
<admiralty-hint
            id={this.hintId}
            disabled={this.disabled}
          >
            {this.hint}
          </admiralty-hint>
```

Same with the `errorId` - we should have something like:

```typescript
  <admiralty-input-invalid
    id={this.errorId}
    style={{ visibility: this.invalid && this.invalidMessage ? 'visible' : 'hidden' }}>. 
      {this.invalidMessage}
</admiralty-input-invalid>
```

Reply:
Done